devtools/conda-envs/asapdiscovery.yaml

```diff
@@ -54,3 +55,6 @@ dependencies:
     - pyyaml
     # ml
     - git+https://github.com/choderalab/mtenn@0.1.0
+    # testing
```

Comment:
These are from #223 ?

Reply:
you added these in https://github.com/choderalab/asapdiscovery/commit/adc1150635b9740bf2a18e753c6157caacd2fe97

Reply:
i added the pebble changes because of #218

Reply:
Ah yep ok.